src/main/scala/com/cognite/sdk/scala/v1/dataModelProperties.scala

```diff
@@ -14,3 +16,40 @@ final case class Float64Property(value: Double) extends PropertyTypePrimitive
 final case class StringProperty(value: String) extends PropertyTypePrimitive
 
 final case class ArrayProperty[+A <: PropertyTypePrimitive](values: Vector[A]) extends PropertyType
+
+final case class TimeStampProperty(value: ZonedDateTime) extends PropertyType
+final case class DateProperty(value: LocalDate) extends PropertyType
+
+// These types below are treated as string for now
+final case class DirectRelationProperty(value: String) extends PropertyType
+final case class GeometryProperty(value: String) extends PropertyType
+final case class GeographyProperty(value: String) extends PropertyType
+
+object PropertyName {
+  val boolean = "boolean"
+  val int32 = "int32"
+  val int64 = "int64"
+  val int = "int"
+  val bigint = "bigint"
+  val float32 = "float32"
+  val float64 = "float64"
+  val numeric = "numeric"
+  val text = "text"
+  val timestamp = "timestamp"
+  val date = "date"
```

Comment:
Nice! Constants like this should be in upper camel case (i.e Boolean, Int64) according to the naming conventions, https://docs.scala-lang.org/style/naming-conventions.html + ctr+f + `Constants, Values and Variables`.

Reply:
Thanks hehe, I updated with the renaming 😁 
Btw here's the link which pointed directly to the section
https://docs.scala-lang.org/style/naming-conventions.html#constants-values-and-variables 😂 , there's a menu on the right side you just have to click to the section ahihihi

Reply:
nnnnice find, I only like the ones that have little § to the left of the headers, like this one https://github.com/cognitedata/cdp-spark-datasource/#read-and-write-to-cdf